torch/storage.py

```diff
@@ -826,6 +859,13 @@ def hpu(self, device=None, non_blocking=False, **kwargs) -> T:  # type: ignore[m
         hpu_storage: torch.UntypedStorage = self._untyped_storage.hpu(device, non_blocking, **kwargs)
         return self._new_wrapped_storage(hpu_storage)
 
+    def to(self, device: torch.device, non_blocking: bool = False, **kwargs) -> T:  # type: ignore[type-var]
```

Comment:
```suggestion
    def to(self, *, device: torch.device, non_blocking: bool = False) -> T:  # type: ignore[type-var]
```

I don't think we need the arbitrary kwargs here as this is a new API.
Using kwargs-only will allow us to add more arguments if needed in the future without breaking BC

Reply:
done